client/state/partner-portal/stored-cards/reducer.ts

```diff
@@ -45,8 +45,45 @@ export const isFetching: Reducer< boolean, IsFetchingAction > = ( state = false,
 	return state;
 };
 
+type IsDeletingAction =
```

Comment:
I'm not 100% convinced about the name for this type. Could something like `DeletingActionStatus` be clearer? 

Honestly, I don't have a strong opinion on this, but I don't think I ever saw a name representing a boolean state for a type.

Reply:
Thanks! I agree with this, I'll change this in the second PR for removing primary payment methods.